server/Arcadia.Assistant.Calendar/Vacations/Events/VacationIsApprovedOnce.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Arcadia.Assistant.Calendar.Vacations.Events
+{
+    using System;
+    using System.Runtime.Serialization;
+
+    [DataContract]
+    public class VacationIsApprovedOnce
+    {
+        [DataMember]
+        public string EventId { get; set; }
+
+        [DataMember]
+        public DateTimeOffset TimeStamp { get; set; }
+
+        [DataMember]
+        public string UserId { get; set; }
```

Comment:
I don't think we put UserId anywhere. It is employee id, right? So it should be ApproverId

Reply:
Done.